paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -707,6 +707,43 @@ class PaparazziPluginTest {
     assertThat(snapshots[2]).isSimilarTo(verticalScroll).withDefaultThreshold()
   }
 
+  @Test
+  fun deleteSnapshots() {
+    val fixtureRoot = File("src/test/projects/delete-snapshots")
+
+    gradleRunner
+      .withArguments("deletePaparazziSnapshots", "--stacktrace")
+      .runFixture(fixtureRoot) { build() }
+
+    val snapshotsDir = File(fixtureRoot, "src/test/snapshots").registerForDeletionOnExit()
+
+    val snapshot = File(snapshotsDir, "images/app.cash.paparazzi.plugin.test_DeleteTest_delete.png")
+    assertThat(snapshot.exists()).isFalse()
+
+    val snapshotWithKeep = File(snapshotsDir, "images/app.cash.paparazzi.plugin.test_DeleteTest_delete_keep.png")
+    assertThat(snapshotWithKeep.exists()).isFalse()
```

Comment:
move the file references to before the runner and add assertions that both existed prior to deletion?